File: C# Fundamentals/C# OOP Basics/03. Defining Classes/Lab/04. Person Class/Person.cs
using System.Collections.Generic;
using System.Linq;

public class Person
{
    private string name;
    private int age;
    private List<BankAccount> accounts;

    public decimal GetBalance()
    {
        return this.accounts.Sum(x => x.Balance);
    }
    public Person(string name, int age, List<BankAccount> accounts)
    {
        this.name = name;
        this.age = age;
        this.accounts = accounts;
    }

    public Person(string name, int age)
    {
        this.name = name;
        this.age = age;
        this.accounts = new List<BankAccount>();
    }

    public string Name
    {
        get
        {
            return this.name;
        }
        set
        {
            this.name = value;
        }
    }
    public int Age
    {
        get
        {
            return this.age;
        }
        set
        {
            this.age = value;
        }
    }
}

File: Java/01.Advanced/01.Advanced/11.Classes/Ex/src/t_01_opinion_poll/Person.java
package t_01_opinion_poll;

public class Person implements Comparable<Person> {
    final String name;
    final int age;

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    @Override
    public int compareTo(Person other) {
        return this.name.compareTo(other.name);
    }
}
